### DISCUSSION TITLE
Place to see working cameras.

### DISCUSSION THREAD

**Discussion by doutorinfamous** [blakeblackshear/frigate#3252] [General]
Hi everybody, is there a wiki or page that shows all the cameras that can work with frigate and home assisntant?

Thnks in advanced

**Comment by NickM-27** [comment]
In general any camera that supports RTSP / HTTP streams can work with frigate, but many have the potential for issues based on their implementation (avoid reolink for this purpose).

The docs have a good starting point for recommendations: https://docs.frigate.video/hardware

If you decide on a camera that isn't there, I'd recommend checking the issues (open and closed) just to see if any users have had issues with that camera.

**Reply by doutorinfamous** [reply]
thanks for the info!! :)
